api/edge/v1alpha1/types.go

```diff
@@ -171,6 +173,125 @@ type PlacementList struct {
 	Items           []Placement `json:"items"`
 }
 
+/******************** End Placement Type *********************/
+
+/******************** Start PlacementDecision Type *********************/
+
+// PlacementDecision is mapped 1:1 to a single Placement object.
+// The decision resource reflects the resolution of the Placement's selectors,
+// and explicitly reflects which objects should go to what destinations.
+//
+// +genclient
+// +genclient:nonNamespaced
+// +kubebuilder:object:root=true
+// +kubebuilder:resource:scope=Cluster,shortName={pd}
+type PlacementDecision struct {
+	metav1.TypeMeta `json:",inline"`
+	// Standard object metadata.
+	// More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+	// +optional
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// `spec` explicitly describes a desired binding between workloads and Locations.
+	// It reflects the resolution of a Placement's selectors.
+	Spec PlacementDecisionSpec `json:"spec,omitempty"`
+
+	// `status` describes the status of the process of binding workloads to Locations.
+	// +optional
+	Status PlacementDecisionStatus `json:"status,omitempty"`
+}
+
+// PlacementDecisionSpec holds a list of objects and a list of destinations which are the resolution
+// of a Placement's `what` and `where`: what objects to propagate and to where.
+// All objects present in this spec are propagated to all destinations present.
+type PlacementDecisionSpec struct {
+	// `Workload` is a collection of namespaced-scoped objects and a collection of cluster-scoped objects to be propagated to destination clusters.
+	Workload DownsyncObjectsReferences `json:"workload,omitempty"`
+
+	// `destinations` is a list of cluster-identifiers that the objects should be propagated to.
+	Destinations []Destination `json:"destinations,omitempty"`
+}
+
+// DownsyncObjectsReferences explicitly defines the objects to be down-synced.
+// The ClusterScope list defines the cluster-scope objects, while NamespacedObjects packs individual objects
+// identifiable by namespace & name.
+type DownsyncObjectsReferences struct {
```

Comment:
We do not need two plurals. I think we want "DownsyncObjectReferences".

Reply:
done